lib/gold_miner/container.rb

```diff
@@ -0,0 +1,3 @@
+module GoldMiner
+  Container = Data.define(:gold_nuggets, :origin, :packing_date)
```

Comment:
Should I make this more explicit?
```suggestion
  GoldContainer = Data.define(:gold_nuggets, :origin, :packing_date)
```